dts/arm/st/f4/stm32f405.dtsi

```diff
@@ -175,6 +175,38 @@
 			status = "disabled";
 			label= "OTGHS";
 		};
+
+		can1: can@40006400 {
+			compatible = "st,stm32-can";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40006400 0x400>;
+			interrupts = <19 0>, <20 0>, <21 0>, <22 0>;
+			interrupt-names = "tx", "rx0", "rx1", "sce";
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x02000000>;
+			status = "disabled";
+			label = "CAN_1";
+			bus-speed = <500000>;
```

Comment:
The stm32f0 uses 125000 here. I think we should agree on one bit rate and apply that to all SoC.

Reply:
I am not sure if things like bitrate belong in DT , because they are application depended and not hardware depended. And if it is just a default-setting it is useless any way because every application will need to make sure the right bitrate is selected anyway (500k in my case).

Of course it is also done by uart DTs so not sure what is the right way to go here. 

I was always thinking DT describes the hardware not the user preferences.

Reply:
@lowlander you can change the bitrate by applying dts overlays. Same for the other settings.